### ISSUE TITLE
DataStore.getClaim(long id) doesn't return subclaims

### ISSUE THREAD

**Issue by EtienneDx** [GriefPrevention/GriefPrevention#814]
### What happened:
I can't get a subclaim by it's id.

### What was expected:
Get the subclaim with it's id

### Steps to reproduce:
Another plugin using getClaim with a subdivision claim id

### Server and GriefPrevention version:
Spigot 1.15.2, latest master (093cbb0)

### Suggestion of fix
Iterating over each subclaim while iterating over the claims to find the corresponding one.

Current code:

https://github.com/TechFortress/GriefPrevention/blob/093cbb0e93b3ce33c2071efcc0e2bc438a0c7d46/src/main/java/me/ryanhamshire/GriefPrevention/DataStore.java#L747-L755

I'll fix it into my fork, it's gonna get added into #811 (I can make it another PR too, I just had the bad idea of staying on my master branch over there so yeah...)

**Comment by RoboMWM** [comment]
is this method for getClaim ever called anywhere else in GP?

**Comment by EtienneDx** [comment]
Only once in the FlatDataStore, to set the parents from the childrens:
https://github.com/TechFortress/GriefPrevention/blob/64e101c3bf79bdb93d9626e34b2263a8292d8f40/src/main/java/me/ryanhamshire/GriefPrevention/FlatFileDataStore.java#L463-L471

**Comment by RoboMWM** [comment]
hmm ok, so that call only expects to get parent claims.

**Comment by EtienneDx** [comment]
Well, yes and no, it expects to get patent claims but it never tries to get a subclaim, so letting it return subclaims shouldn’t cause any problems (and it doesn’t on my test server)

**Comment by RoboMWM** [comment]
I mean the call in FlatFileDataStore

**Comment by EtienneDx** [comment]
me too

**Comment by EtienneDx** [comment]
It tries and get the parent by id, but its parent id is unique so there won't be any clashes between parent claims and subclaims
